io.github.Fndroid.clash_for_windows.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: io.github.Fndroid.clash_for_windows
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+base: org.electronjs.Electron2.BaseApp
+base-version: 21.08
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: cfw.sh
+
+finish-args:
+- --share=ipc
+- --share=network
+- --socket=x11
+- --device=dri
+- --filesystem=xdg-config/clash:create
+- --talk-name=org.freedesktop.Notifications
+- --talk-name=org.kde.StatusNotifierWatcher
+
+modules:
+  - name: clash_for_windows
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 apply_extra /app/bin/
+      - install -Dm755 cfw.sh /app/bin/
+      - install -Dm644 io.github.Fndroid.clash_for_windows.png /app/share/icons/hicolor/256x256/apps/io.github.Fndroid.clash_for_windows.png
+      - install -Dm644 io.github.Fndroid.clash_for_windows.desktop /app/share/applications/io.github.Fndroid.clash_for_windows.desktop
+      - install -Dm644 io.github.Fndroid.clash_for_windows.appdata.xml /app/share/metainfo/io.github.Fndroid.clash_for_windows.appdata.xml
+
+    sources:
+      - type: extra-data
+        filename: clash_for_windows.tar.gz
+        url: https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.19.4/Clash.for.Windows-0.19.4-x64-linux.tar.gz
```

Comment:
Consider configuring https://github.com/flathub/flatpak-external-data-checker/ to automate updates.

Reply:
> Consider configuring https://github.com/flathub/flatpak-external-data-checker/ to automate updates.

hello, I have add `x-checker-data` throuth [this commit](https://github.com/qvshuo/flathub/commit/6914d94e4adc464af7f22c0aab06edbce141f8a1)